Game/player_board.gd

```diff
@@ -0,0 +1,35 @@
+extends VBoxContainer
+
+@export var MineMap : TileMapLayer
+@export var FactoryMap : TileMapLayer
+
+# multiplayer properties
+@export var OwnerId : int
+@export var OwnerName : String
+
+# game board properties
+@export var NumCols : int = 30
+@export var LayerThickness : int = 10
+@export var NumMineLayers : int = 1
+@export var SkyHeight : int = 100
+@export var TileMapScale : int = 2
+
+func _ready() -> void:
+	print("doing ready for %s (%s)" % [OwnerName, OwnerId])
+
+	var tile_size := 16 * TileMapScale
+	var board_width_px := tile_size * NumCols
+	var layer_height_px := tile_size * LayerThickness
+
+	custom_minimum_size = Vector2i(board_width_px, 0)
+	$Sky.custom_minimum_size = Vector2i(0, SkyHeight)
```

Comment:
Avoid $ references directly in code. Prefer % references and do them at the top of code, assigning them to a variable in the module